build.gradle

```diff
@@ -3,32 +3,22 @@ plugins {
 }
 
 project.ext.springdoc = '1.2.32'
+project.ext.jupiter = '5.6.0'
```

Comment:
Since you're using boot, you could skip everything except springdoc and leave in inlined, then dependabot would update that. Other versions provided by spring